content/tutorials/authenticate-user-wazo-api.md

```diff
@@ -0,0 +1,153 @@
+Title: How to Authenticate a User From Wazo APIs
+Date: 2022-06-22 10:00:00
+Author: Pascal Cadotte Michaud
+Category: Developper's Tutorial
+Slug: authenticate-user-wazo-api
+OgImage: authenticate-user-wazo-api-og.jpg
+Thumbnail: authenticate-user-wazo-api-thumbnail.jpg
+Status: published
+
+> Original content comes from [Refresh Token in Wazo](/blog/wazo-auth-refresh-token) blog post.
```

Comment:
Should we mention that?

I can also make a redirect from the blog post to the tutorial. Content are almost the same.